internal/k8s/kctl/deploy.go

```diff
@@ -48,3 +64,35 @@ func execute(config Config, command string, args ...string) error {
 	cmd.Stderr = config.Stderr()
 	return cmd.Run()
 }
+
+////// Name Spaces
```

Comment:
This comment can be avoided. Seems like code smell that points to creation of a new file `namespace.go`